### PR TITLE
Feature-979 - JSON-Lake Function

### PR DESCRIPTION
**Pull request by guntukuhari**

### CODE REVIEW

File: fns-hl7-pipeline/fn-hl7-json-lake/src/main/kotlin/gov/cdc/dex/hl7/model/Segment.kt
Comment (mscaldas2012) [inline fns-hl7-pipeline/fn-hl7-json-lake/src/main/kotlin/gov/cdc/dex/hl7/model/Segment.kt:1]: is this class being used for this function?

File: fns-hl7-pipeline/fn-hl7-json-lake/README.md
Comment (mscaldas2012) [inline fns-hl7-pipeline/fn-hl7-json-lake/README.md:1]: This readme seems to belong to a different FN - copy & paste issues?

Comment (guntukuhari) [reply fns-hl7-pipeline/fn-hl7-json-lake/README.md:1]: Yes. Updated, but needs to update more. Please feel free to provide what needs to add

File: fns-hl7-pipeline/fn-hl7-json-lake/src/main/kotlin/gov/cdc/dex/hl7/model/Segment.kt
Comment (guntukuhari) [reply fns-hl7-pipeline/fn-hl7-json-lake/src/main/kotlin/gov/cdc/dex/hl7/model/Segment.kt:1]: No. Deleted

**Review by mscaldas2012 — APPROVED** [review]

**Pull request merged**